site/README.md

```diff
@@ -15,65 +15,94 @@ Read how to edit files for github ([GitHub docs](https://docs.github.com/en/gith
 Please note that files have a markup for correct display on the site: the title, the title of the link,
 the weight (affects the order of files display on the sidebar) and description (optional):
 
-    ---
-    title: "Title"
-    linkTitle: "Link Title"
-    weight: 1
-    description: >
-        Description
-    ---
+```
+---
+title: "Title"
+linkTitle: "Link Title"
+weight: 1
+description: >
+    Description
+---
+```
 
-### Start site localy
 
-To start the site locally, you need a recent [extended version hugo](https://github.com/gohugoio/hugo/releases)
-(recommend version 0.83.1).
-Open the most recent release and scroll down until you find a list of Extended versions. [Read more](https://gohugo.io/getting-started/installing/#quick-install)
+### How to start the site locally
 
-Add a path to "hugo" in the "Path" environment variable.
+#### Installation
+
+1. Get the code
```

Comment:
Does getting the code really count as installation? IMO, it should be spun off into its own section, or the section should be renamed to something more generic (e.g. "Preparation").

Reply:
This submodule is not downloaded by default in the development environment setup instructions, so it may be missing, and so it makes sense to include this info here. 

I don't see how renaming the section could help with understanding the section - installation is a broad topic, and can include different actions.